src/entity/NotificationAvatar.ts

```diff
@@ -0,0 +1,33 @@
+import { Column, Entity, Index, ManyToOne, PrimaryColumn } from 'typeorm';
+import { Notification } from './Notification';
+
+export type NotificationAvatarType = 'source' | 'user';
```

Comment:
Same question here, for instance with spammers who get deleted and perhaps use inappropriate images?

Reply:
Here we already store the `referenceId`